examples/Mediawiki/MoveAPI.yaml

```diff
@@ -0,0 +1,162 @@
+            title: Talk:MoveWithoutt
+            createonly: true
+            format: json
+          form-data:
+            summary: create page
+            text: Talk page for Move withoutt
+            token: +\
+          response:
+            body:
+              edit:
+                result: Success
+      # move page without redirect, subpage, and talkpage
+      - request:
+          method: post
+          path: api.php
+          parameters:
+            action: move
+            from: MoveWithoutt
+            to: Move without
+            noredirect: ""
+            format: json
+          form-data:
+            reason: misspelling
+            token: +\
+        response:
+          body:
+            move:
+              from: MoveWithoutt
+              to: Move without
+              reason: misspelling
+              redirectcreated: ""
+  - description: Create page and move with redirect, subpage, and talkpage
+    interaction:
+      # create page
+      - request:
+          method: post
+          path: api.php
+          parameters:
+            action: edit
+            title: MoveWithh
+            createonly: true
+            format: json
+          form-data:
+            summary: create page
+            text: Move with redirect, subpage and talkpage
+            token: +\
+        response:
+          body:
+            edit:
+              result: Success
+      # create subpage
+      - request:
+          method: post
+          path: api.php
+          parameters:
+            action: edit
+            title: MoveWithh/subpage
+            createonly: true
+            format: json
+          form-data:
+            summary: create page
+            text: Subpage of MoveWith
+            token: +\
+          response:
+            body:
+              edit:
+                result: Success
+      # create talk page
+      - request:
+          method: post
+          path: api.php
+          parameters:
+            action: edit
+            title: Talk:MoveWithh
+            createonly: true
+            format: json
+          form-data:
+            summary: create page
+            text: Talk page for MoveWith
+            token: +\
+          response:
+            body:
+              edit:
+                result: Success
+      # move page with redirect, subpage, and talkpage
+      - request:
+          method: post
+          path: api.php
+          parameters:
+            action: move
+            from: MoveWithh
+            to: Move With
+            movetalk: ""
+            movesubpages: ""
+            format: json
+          form-data:
+            reason: misspelling
+            token: +\
+        response:
+          body:
+            move:
+              from: MoveWithh
+              to: Move With
+              reason: misspelling
+              talkfrom: Talk:MoveWithh
+              talkto: Talk:Move With
+              subpagefrom: MoveWithh
+              subpageto: Move With
```

Comment:
is that really the response format? i didn't check, but... how would that work if multiple subpages get moved?

Reply:
No, it's mostly a place holder.